impl_core/src/main/java/io/opencensus/implcore/trace/RecordEventsSpanImpl.java

```diff
@@ -586,15 +586,4 @@ private RecordEventsSpanImpl(
         timestampConverter != null ? timestampConverter : TimestampConverter.now(clock);
     startNanoTime = clock.nowNanos();
   }
-
-  @SuppressWarnings("NoFinalizer")
```

Comment:
Please see original PR #1877 which fixed the issue #1876 by adding this code.
As you can see, this code change was deliberate, so removing it would probably require addressing the issue differently.

Reply:
Given the severity of the issue this code is causing, we decided to revisit the decision and proceed with this change. See #2045 for more context.